src/Controller/Speaker/Create.php

```diff
@@ -48,7 +48,13 @@ public function handle(Request $request): Response
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
-            $speakerRequest->photoPath = $this->fileUploader->upload($speakerRequest->photo);
+            if (!empty($speakerRequest->photo)) {
+                $speakerRequest->photoPath = $this->fileUploader->upload($speakerRequest->photo);
+            }
+
+            if (empty($speakerRequest->photoPath)) {
```

Comment:
You should do that prior the `!empty($speakerRequest->photo)` (as an default value), and then use the fileUploader value if it's filled. It helps with removing ifs, hence lowering cyclomatic complexity.